.github/workflows/regression_tests.yml

```diff
@@ -76,10 +76,10 @@ jobs:
         uses: actions/cache@a2ed59d39b352305bdd2f628719a53b2cc4f9613
         id: cache-venv
         with:
-          path: ./.venv/
-          key: ${{ runner.os }}-${{ steps.setup_python.outputs.python-version }}-venv-${{ hashFiles('pyproject.toml') }}
+          path: .venv
+          key: ${{ runner.os }}-venv-${{ hashFiles('pyproject.toml') }}
```

Comment:
Have these changes been tested?

Reply:
This doesn't work either way. I have a tech-debt ticket to address this: